app/src/main/java/com/example/playlistmaker/presentation/playlist/ShowAlbumFragment.kt

```diff
@@ -0,0 +1,214 @@
+package com.example.playlistmaker.presentation.playlist
+
+import android.content.Intent
+import android.net.Uri
+import android.os.Bundle
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.LinearLayout
+import android.widget.Toast
+import androidx.core.os.bundleOf
+import androidx.navigation.fragment.findNavController
+import androidx.recyclerview.widget.LinearLayoutManager
+import androidx.recyclerview.widget.RecyclerView
+import com.bumptech.glide.Glide
+import com.bumptech.glide.load.resource.bitmap.RoundedCorners
+import com.example.playlistmaker.R
+import com.example.playlistmaker.databinding.FragmentShowAlbumBinding
+import com.example.playlistmaker.domain.models.Album
+import com.example.playlistmaker.domain.models.Track
+import com.example.playlistmaker.presentation.player.PlayerFragment
+import com.example.playlistmaker.presentation.search.MusicAdapter
+import com.example.playlistmaker.presentation.viewmodels.ShowAlbumViewModel
+import com.google.android.material.bottomsheet.BottomSheetBehavior
+import com.google.android.material.dialog.MaterialAlertDialogBuilder
+import org.koin.androidx.viewmodel.ext.android.viewModel
+import java.text.SimpleDateFormat
+import java.util.*
+
+class ShowAlbumFragment : Fragment() {
+
+    private lateinit var recycler: RecyclerView
+    private var _binding: FragmentShowAlbumBinding? = null
+    private var musicAdapter = MusicAdapter()
+    private val binding get() = _binding
```

Comment:
```suggestion
    private val binding get() = _binding!!
```

Иначе толку от заведения такой переменной не много